src/Stratis.Features.FederatedPeg/FederationGatewaySettings.cs

```diff
@@ -28,16 +28,25 @@ public sealed class FederationGatewaySettings : IFederationGatewaySettings
         private const string MinimumDepositConfirmationsParam = "mindepositconfirmations";
 
         /// <summary>
-        /// The transaction fee used by the federation to build withdrawal transactions.
+        /// The fee taken by the federation to build withdrawal transactions. The federation will keep most of this.
         /// </summary>
-        /// <remarks>
-        /// Changing <see cref="TransactionFee"/> affects both the deposit threshold on this chain and the withdrawal transaction fee on this chain.
+        /// Changing <see cref="CrossChainTransferFee"/> affects both the deposit threshold on this chain and the withdrawal transaction fee on this chain.
         /// This value shouldn't be different for the 2 pegged chain nodes or deposits could be extracted that don't have the amount required to
         /// cover the withdrawal fee on the other chain.
         ///
         /// TODO: This should be configurable on the Network level in the future, but individual nodes shouldn't be tweaking it.
         /// </remarks>
-        public static readonly Money DefaultTransactionFee = Money.Coins(0.001m);
+        public static readonly Money CrossChainTransferFee = Money.Coins(0.001m);
```

Comment:
remarks tag above wasn't open (there is `</remarks>` but no `<remarks>` in the beginning)